recipes/rangeless/all/conanfile.py

```diff
@@ -1,34 +1,46 @@
-from conans import ConanFile, tools
 import os
-import glob
+
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
+
+required_conan_version = ">=1.52.0"
+
 
 class RangelessConan(ConanFile):
     name = "rangeless"
     description = "c++ LINQ -like library of higher-order functions for data manipulation"
     license = "MIT"
-    homepage = "https://github.com/ast-al/rangeless"
     url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ast-al/rangeless"
     topics = ("range", "linq", "lazy-evaluation", "header-only")
-    settings = "compiler"
+
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
     no_copy_source = True
 
     @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    def _min_cppstd(self):
+        return 14
 
-    def configure(self):
-        minimal_cpp_standard = "14"
-        if self.settings.compiler.cppstd:
-            tools.check_min_cppstd(self, minimal_cpp_standard)
+    def layout(self):
+        basic_layout(self, src_folder="src")
 
     def package_id(self):
-        self.info.header_only()
+        self.info.clear()
+
+    def validate(self):
+        if self.settings.compiler.cppstd:
```

Comment:
This is missing the `compiler_minimum_version` checks, like for https://github.com/conan-io/conan-center-index/pull/18250/files#diff-100c2c6de539067c6fb843bd683e38979259f7c5e4e7a3e17bd1defa05b5fedeR30

These kinds of checks will be required until v1 is no longer supported in CCI, as they ensure we have something like cppstd checks for v1 (The v1 profiles do not define cppstd, so we use them as a fall-back)